project/gen2.scala

```diff
@@ -120,6 +120,12 @@ object gen2 {
         io.transact(xa).unsafePerformIO
       },
 
+      enum("F2WindowCover") {
+        type F2WindowCoverRec = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'tccValue -> String`.T
+        val io = sql"select id, id tag, short_name, long_name, tcc_value from e_f2_window_cover".query[(String, F2WindowCoverRec)].list
+        io.transact(xa).unsafePerformIO
+      },
```

Comment:
Incidentally the witness hack is no longer required with the Typelevel Scala compiler. You can say `val x: List[42] = ...` for a list that can only contain the value `42`. With some luck this will get merged into mainline Scala in the 2.12 timeframe.